assets/js/script.js

```diff
@@ -0,0 +1,91 @@
+var grid = Array(100).fill(0)
```

Comment:
I gave you inappropriate answer about this, so remain new answer here.
Do you remember `var array = Array(10).fill(Array(10).fill(0))` was acting weird?

```
var array = Array(10).fill(Array(10).fill(0))

var shallowCopied = array.slice()
var deepCopied = array.map(e => e.slice())

array[0][3] = 1

console.log(shallowCopied)
console.log(deepCopied)
```
Try above.

That was because `.slice()` is just shallow copy, inside arrays of 2D were not copied at all. So, when we re-assign value there, our original array was changed.

I hope you google about shallow copy vs deep copy if you want to know more. :)